apps/site/components/EOL/EOLModal/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { Modal, Title, Content } from '@node-core/ui-components/Common/Modal';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import UnknownSeveritySection from '#site/components/EOL/UnknownSeveritySection';
+import VulnerabilitiesTable from '#site/components/EOL/VulnerabilitiesTable';
+import { SEVERITY_ORDER } from '#site/next.constants.mjs';
+import type { ModalProps } from '#site/types';
+import type {
+  EOLModalData,
+  KnownVulnerability,
+  Vulnerability,
+} from '#site/types/vulnerabilities';
+
+const EOLModal: FC<ModalProps> = ({ open, closeModal, data }) => {
+  const { release, vulnerabilities } = data as EOLModalData;
+  const t = useTranslations();
+
+  const modalHeading = t(
+    release.codename
+      ? 'components.eolModal.title'
+      : 'components.eolModal.titleWithoutCodename',
+    {
+      version: release.major,
+      codename: release.codename ?? '',
+    }
+  );
+
+  const [knownVulns, unknownVulns] = vulnerabilities.reduce(
```

Comment:
nit: Remember to memoize these calls.

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7990/commits/847897bf1c234df08bc175e2d94dc39fc4e87299